pkg/project/admission/lifecycle/admission.go

```diff
@@ -91,6 +91,10 @@ func (e *lifecycle) Admit(a admission.Attributes) (err error) {
 
 	namespace, err := projects.GetNamespaceObject(a.GetNamespace())
 	if err != nil {
+		// do not leak that the namespace was not found
+		if apierrors.IsNotFound(err) {
+			return admission.NewForbidden(a, nil)
```

Comment:
@deads2k - do we have a standard not-authorized message that would be appropriate here?

I would expect something like "You are not authorized to perform this action at this time." type of thing.